### PR TITLE
Add new commands to Discord

### PR DESCRIPTION
**Pull request by alexander-clarke**
This adds some new commands to discord, and additionally fixes an existing command.

I have only implemented these commands currently on Windows, as that is the OS that I use.

This is my first contribution to this repository, so do tell me if I'm doing anything wrong.

### CODE REVIEW

File: apps/discord/win.py [hunk 1]
```diff
@@ -82,3 +89,23 @@ def discord_answer_call():
...
+        actions.key("ctrl-k")
+        if dest_type:
+            if dest_type == "user":
+                actions.key("@")
+            if dest_type == "channel":
+                actions.key("#")
+            if dest_type == "voice":
+                actions.key("!")
+            if dest_type == "server":
+                actions.key("*")
```
Comment (auscompgeek) [inline apps/discord/win.py:109]: We could put this mapping in the list above instead.

Comment (alexander-clarke) [reply apps/discord/win.py]: yeah that's a good idea, it's in the list now

File: apps/discord/win.py [hunk 2]
```diff
@@ -7,6 +7,13 @@
 app: discord
 """
 
+ctx.lists["user.discord_destination"] = {
```
Comment (auscompgeek) [inline apps/discord/win.py:10]: Should these be available for macOS too?

Comment (alexander-clarke) [reply apps/discord/win.py]: yeah that's true, for some reason I tried to do this before and it didn't work, but I got it to work this time

File: apps/discord/discord.talon [hunk 3]
```diff
@@ -7,6 +7,10 @@ tag(): user.emoji
 [channel] mentions last: user.discord_mentions_last()
 [channel] mentions next: user.discord_mentions_next()
 oldest unread: user.discord_oldest_unread()
+{user.discord_destination} [<user.text>]: user.discord_quick_switcher(user.discord_destination or "", user.text or "")
```
Comment (auscompgeek) [inline apps/discord/discord.talon:10]:
```suggestion
{user.discord_destination} [<user.text>]: user.discord_quick_switcher(user.discord_destination, user.text or "")
```

**Comment by alexander-clarke** [comment]
I just fixed the merge issues related to some of the bigger commits recently

**Review by splondike — APPROVED** [review]
Thanks!

**Pull request merged**